src/rpcmisc.cpp

```diff
@@ -172,6 +172,12 @@ UniValue mnsync(const UniValue& params, bool fHelp)
         return obj;
     }
 
+    if(strMode == "next")
+    {
+        masternodeSync.GetNextAsset();
+        return "sync updated to " + masternodeSync.GetAssetName();
```

Comment:
Nice idea to make id more human readable :)
Probably a good idea to add `obj.push_back(Pair("RequestedMasternodeAssets", masternodeSync.GetAssetName()));` in `status` output too.